oracle/datadog_checks/oracle/oracle.py

```diff
@@ -160,7 +160,7 @@ def _get_custom_metrics(self, con, custom_queries, global_tags):
 
             with closing(con.cursor()) as cursor:
                 cursor.execute(query)
-                for row in cursor:
+                for row in cursor.fetchall():
```

Comment:
Let's batch results instead of reading everything into memory with `fetchmany`

Reply:
We use fetchall everywhere else in the check, do you mind if we do that later?

Reply:
Sure!